updated docs (#31192)
demisto/content: Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/README.md

### msgraph-get-managed-device-by-id
***
Get managed devices

>  - Please notice that by default physicalMemoryInBytes will return 0 since Microsoft haven't updated the v1.0 schema with that information yet.

Reviewer: ```suggestion
 -Note: physicalMemoryInBytes will return 0 by default as Microsoft v1.0 schema is not updated with that information.
```
